src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/TreeView/TreeViewPage.xaml.cs

```diff
@@ -0,0 +1,1084 @@
+			if (TestTreeView.SelectionMode == TreeViewSelectionMode.Single)
+			{
+				selectionMode = TreeViewSelectionMode.Multiple;
+			}
+			else if (TestTreeView.SelectionMode == TreeViewSelectionMode.Multiple)
+			{
+				selectionMode = TreeViewSelectionMode.None;
+			}
+			else
+			{
+				selectionMode = TreeViewSelectionMode.Single;
+			}
+
+			TestTreeView.SelectionMode = selectionMode;
+			ContentModeTestTreeView.SelectionMode = selectionMode;
+		}
+
+		private void GetItemCommonStates_Click(object sender, RoutedEventArgs e)
+		{
+			string commonStates = string.Empty;
+			var listControl = FindVisualChildByName(this.TestTreeView, "ListControl") as TreeViewList;
+			if (IsInContentMode())
+			{
+				listControl = FindVisualChildByName(ContentModeTestTreeView, "ListControl") as TreeViewList;
+			}
+
+			for (int i = 0; i < GetItemsCount(listControl); i++)
+			{
+				var container = (TreeViewItem)listControl.ContainerFromIndex(0);
+				var groups = VisualStateManager.GetVisualStateGroups((FrameworkElement)VisualTreeHelper.GetChild(container, 0));
+				foreach (var group in groups)
+				{
+					if (group.Name == "CommonStates")
+					{
+						commonStates += group.CurrentState.Name + " ";
+					}
+				}
+			}
+
+			Results.Text = commonStates;
+		}
+
+		private void TestTreeView_Expanding(TreeView sender, TreeViewExpandingEventArgs args)
+		{
+			var loadingNode = args.Node;
+			if (loadingNode == _virtualizedNode)
+			{
+				loadingNode.Children.Add(new TreeViewNode() { Content = "Loaded: 1" });
+				loadingNode.Children.Add(new TreeViewNode() { Content = "Loaded: 2" });
+				loadingNode.Children.Add(new TreeViewNode() { Content = "Loaded: 3" });
+				loadingNode.HasUnrealizedChildren = false;
+
+				Results.Text = "Loaded";
+			}
+		}
+
+		private void ContentModeTestTreeView_Expanding(TreeView sender, TreeViewExpandingEventArgs args)
+		{
+			var loadingItem = args.Item as TreeViewItemSource;
+			var count = TestTreeViewItemsSource[0].Children.Count;
+			if (loadingItem == TestTreeViewItemsSource[0].Children[count - 1])
+			{
+				loadingItem.Children.Add(new TreeViewItemSource() { Content = "Loaded: 1" });
+				loadingItem.Children.Add(new TreeViewItemSource() { Content = "Loaded: 2" });
+				loadingItem.Children.Add(new TreeViewItemSource() { Content = "Loaded: 3" });
+				loadingItem.HasUnrealizedChildren = false;
+
+				Results.Text = "Loaded";
+			}
+		}
+
+		private void ChangeFlowDirection_Click(object sender, RoutedEventArgs e)
+		{
+			TestTreeView.FlowDirection = FlowDirection.RightToLeft;
+			ContentModeTestTreeView.FlowDirection = FlowDirection.RightToLeft;
+		}
+
+		private void GetItemCount_Click(object sender, RoutedEventArgs e)
+		{
+			var listControl = FindVisualChildByName(TestTreeView, "ListControl") as TreeViewList;
+			if (IsInContentMode())
+			{
+				listControl = FindVisualChildByName(ContentModeTestTreeView, "ListControl") as TreeViewList;
+			}
+			Results.Text = GetItemsCount(listControl).ToString();
```

Comment:
Codacy found an issue: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)